### ISSUE TITLE
ScriptError: Name (filename, int, int)  is defined twice, at (filename:line) and (filename:line)

### ISSUE THREAD

**Issue by maLikiz** [renpy/renpy#3393]
When I move my part of the code above I get an error:
```
I'm sorry, but an uncaught exception occurred.

While parsing C:\renpy-7.4.11-sdk\WitchHunter/game/scripts/fight/fight.rpy.
ScriptError: Name (u'game/scripts/fight/fight.rpy', 1639898863, 1911) is defined twice, at game/scripts/fight/fight.rpy:362 and game/scripts/fight/fight.rpy:365.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "renpy/main.py", line 492, in main
    renpy.game.script.load_script() # sets renpy.game.script.
  File "renpy/script.py", line 283, in load_script
    self.load_appropriate_file(".rpyc", ".rpy", dir, fn, initcode)
  File "renpy/script.py", line 797, in load_appropriate_file
    self.finish_load(stmts, initcode, filename=lastfn)
  File "renpy/script.py", line 473, in finish_load
    check_name(node)
  File "renpy/script.py", line 452, in check_name
    bad_node.filename, bad_node.linenumber))
ScriptError: Name (u'game/scripts/fight/fight.rpy', 1639898863, 1911) is defined twice, at game/scripts/fight/fight.rpy:362 and game/scripts/fight/fight.rpy:365.

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
 
Wed Apr  6 17:50:51 2022

```

It seems to me that this problem arises because I started using the `old-game`

My changes:
![image](https://user-images.githubusercontent.com/4878139/161979959-30102af0-6299-4484-bc7e-747e2eaa46cb.png)

**Comment by renpytom** [comment]
Did you put old-game inside game, or at a higher level? It  should be at the same level as game, not inside it.

**Comment by Andykl** [comment]
I had the exact same problem, wanted to report it but still have no reliable way to reproduce it. And I put old-game correctly for sure.
This happens only if you have both rpyc files in `old-game` and in `game` folder and change the script (always or almost always this happened when moving the code block as in the screenshot above). So, deleting any of rpyc files fixes the problem.
It happens not each time, so I suppose it something related to how SequenceMatcher works.

**Comment by renpytom** [comment]
If you can zip the files up and send them to me, that would work, as I could debug things.

**Comment by maLikiz** [comment]
I have this file added to the `old-game` and to the `game` folder, I don't remember exactly if there was a difference in them before this change. Unfortunately I don't have the old `.rpyc` file from the `game` folder. If I reproduce this error again in the future, I will definitely send you files from the `game` and `old-game` folders.

**Comment by Andykl** [comment]
@maLikiz I found the bug. It is fixed in next nighly and 7.5/8.0 when it is out.

**Comment by maLikiz** [comment]
I caught this error again on the version 7.4.11.2266, rpy and rpyc files:
[fight.zip](https://github.com/renpy/renpy/files/8790893/fight.zip)
